DependencyInjection/GraphQLiteCompilerPass.php

```diff
@@ -4,10 +4,12 @@
 namespace TheCodingMachine\GraphQLite\Bundle\DependencyInjection;
 
 use Doctrine\Common\Annotations\PsrCachedReader;
+use Generator;
 use GraphQL\Server\ServerConfig;
 use GraphQL\Validator\Rules\DisableIntrospection;
 use GraphQL\Validator\Rules\QueryComplexity;
 use GraphQL\Validator\Rules\QueryDepth;
+use Kcs\ClassFinder\Finder\ComposerFinder;
```

Comment:
I do not know this library (yet). The code look cleaner with it.

Were there other such libraries considered?

Reply:
we discussed it here https://github.com/thecodingmachine/graphqlite/issues/657